### DISCUSSION TITLE
API  for retreiving all document initiated per user

### DISCUSSION THREAD

**Discussion by dilshadMNS** [docusealco/docuseal#368] [Q&A] [answered]
Hello Team,
I am planning to get on premise docuseal pro and I have an angular web application that connects to the docuseal and use the embed form for document signing. The user is registered on my application and user login my application and via my application  connects to docuseal perform self sign and also initiate document to be sign by other parties.
My question is since my  web application is connecting to the docuseal pro and not the user,is there a REST endpoint that retreive a list of all document that was initiated by the user ?Is there also an API that retreive list of document that is not signed yet which was initiated by a specific user registered on my application?

Thanks in advance.

**Comment by omohokcoj** [comment, marked as answer]
@dilshadMNS currently we don't have that API filter param - but probably for your use case it would be best to use webhooks (submission.created) https://www.docuseal.co/docs/api#submission-webhook - this webhook returns the "created_by_user" which you can store in your application database - so that you can load/filter all submissions directly in your app to build an API for your angular frontend.

**Reply by dilshadMNS** [reply]
Thanks Pete
